CHANGES.md

```diff
@@ -2,25 +2,15 @@ Changes by Version
 ==================
 Release Notes.
 
-0.1.0
+0.2.0
```

Comment:
Where do you put the previous changelog?

Reply:
https://github.com/apache/skywalking-satellite/blob/v0.1.0/changes/changes-0.1.0.md

Reply:
We usually keep all versions' changelogs. Check the main repo.

Reply:
yes, already keeps it. I don't remove it.

Reply:
https://github.com/apache/skywalking-satellite/blob/fix-race-bug/changes/changes-0.1.0.md